diff --git a/dist/node/chunks/dep-DXWVQosX.js b/dist/node/chunks/dep-DXWVQosX.js
index dbed6118fec170112dfc4b9b8dda74a30dfd7cdf..ef01804785ab0f771374ab85183e849d9fd79179 100644
--- a/dist/node/chunks/dep-DXWVQosX.js
+++ b/dist/node/chunks/dep-DXWVQosX.js
@@ -55260,9 +55260,9 @@ async function fetchModule(environment, url, importer, options = {}) {
     }
     if (url[0] !== '.' && url[0] !== '/') {
         const { isProduction, root } = environment.config;
-        const { externalConditions, dedupe, preserveSymlinks } = environment.options.resolve;
+        const { externalConditions, dedupe, preserveSymlinks, mainFields, extensions } = environment.options.resolve;
         const resolved = tryNodeResolve(url, importer, {
-            mainFields: ['main'],
+            mainFields,
             conditions: [],
             externalConditions,
             external: [],
@@ -55272,7 +55272,7 @@ async function fetchModule(environment, url, importer, options = {}) {
                 'production',
                 'development',
             ],
-            extensions: ['.js', '.cjs', '.json'],
+            extensions,
             dedupe,
             preserveSymlinks,
             isBuild: false,
